internal/webhook/core/v1/pod_webhook_test.go

```diff
@@ -63,8 +65,41 @@ var _ = Describe("Pod Mutator Webhook", func() {
 			Expect(multinodePod.Spec.InitContainers).ShouldNot(BeNil())
 			Expect(multinodePod.Spec.InitContainers).Should(HaveLen(1))
 			Expect(multinodePod.Spec.InitContainers[0].Name).To(Equal(constants.RayTLSGeneratorInitContainerName))
+
+			// Verify that the volumes were added
+			Expect(multinodePod.Spec.Volumes).ShouldNot(BeNil())
+			foundTLS := false
+			foundTLSSecret := false
+
+			for _, volume := range multinodePod.Spec.Volumes {
+				if volume.Name == constants.RayTLSVolumeName {
+					foundTLS = true
+				}
+				if volume.Name == constants.RayTLSSecretVolumeName {
+					foundTLSSecret = true
+				}
+
+				if foundTLS && foundTLSSecret {
+					break
+				}
+			}
+
+			Expect(foundTLS).To(BeTrue(), fmt.Sprintf("Expected volume %s to be present", constants.RayTLSVolumeName))
+			Expect(foundTLSSecret).To(BeTrue(), fmt.Sprintf("Expected volume %s to be present", constants.RayTLSSecretVolumeName))
```

Comment:
May the following work?

```go
Expect(multinodePod.Spec.Volumes).Should(ContainElements(
  HaveField("Name", constants.RayTLSVolumeName),
  HaveField("Name", constants.RayTLSSecretVolumeName)
))
```

... so that you don't need the loop/search nor the auxiliary variables.

Reply:
thx updated!